setup.cfg

```diff
@@ -36,7 +36,7 @@ install_requires =
 
 [options.extras_require]
 tests =
-    pytest-black>=0.3.0,<0.3.10
+    pytest-black>=0.3.0
```

Comment:
here we are probably missing to depend on the newly released version of `invenio-i18n`: I guess that otherwise the import `from invenio_i18n import lazy_gettext as _` will fail?

Reply:
the invenio-i18n is in the install_requires section. should we move it to tests?

Reply:
You are totally right, I am sorry.. I didn't see it in the diff and I didn't check, my bad! :)